### ISSUE TITLE
[Feature] Ability to update font, fontsize, row padding, alternating row color

### ISSUE THREAD

**Issue by shirshak55** [ProxymanApp/Proxyman#255]
This idea come from tableplus editor. 

<img width="560" alt="image" src="https://user-images.githubusercontent.com/8097377/63220401-58324e80-c1a4-11e9-9dc9-c834cda3c160.png">

**Comment by NghiaTranUIT** [comment]
In the latest build, 2.15.1, we can adjust the font size and select mono-font for the body data 🎉 

<img width="702" alt="Screen Shot 2020-12-25 at 16 34 50" src="https://user-images.githubusercontent.com/5878421/103130397-1f2ab800-46cf-11eb-93a0-7104e3f48fbc.png">

**Comment by webuniverseio** [comment]
Hello, is it possible to add font customization of the UI? On 5k retina display everything looks very small. To start even just adding ability to change font for UI at my own risk would be good enough for me and if there are specific issues I can report them separately.

**Comment by webuniverseio** [comment]
Current problem is that only parts which are showing in main window are customizable but all tools, filters etc is very small. Thank you.

**Comment by NghiaTranUIT** [comment]
Sorry if it causes trouble @webuniverseio 

For the entire UI, customizing the font size/font name might terribly break the app UI.

However, I can support Font Size Customization on all debugging Tools (Only the Table View View). Let me know if it's good, then I can fix it :+1

**Comment by webuniverseio** [comment]
Hey @NghiaTranUIT no worries, there are various apps which are not ideal for retina on desktop but I can't go back to blurry/pixelated screens.

I'm thinking - would it be helpful if I create a bunch of tickets with specific parts which I would want to have an increased size? Or better to put it all in one story with details/screenshots? I understand that addressing it throughout entire app will be tricky, so I'm not asking for that. Just trying to think about best way to provide feedback for parts that are too small.

Thanks

**Comment by NghiaTranUIT** [comment]
Yes, it's possible to create multiple small tickets to enhance accessibility. If you don't mind, which part of the app you'd need to support a bigger Font Size? The feature that you've used the most.

**Comment by webuniverseio** [comment]
Honestly right now this is across multiple areas. Scripting (code/console font sizes increase would be nice), map remote/local. I use other areas like black/white lists. Maybe it make sense to add support for different display size variants/settings - normal/hi-resolution to have a predictable settings for UI.

**Comment by NghiaTranUIT** [comment]
> Honestly right now this is across multiple areas. Scripting (code/console font sizes increase would be nice), map remote/local. I use other areas like black/white lists. Maybe it make sense to add support for different display size variants/settings - normal/hi-resolution to have a predictable settings for UI.

I still prefer "Keep it simple" by using 1 config in the Preference. 

I mean, the font size will affect all UI of the app, including the Main Table View, Request/Response, Tool Rule Table View, Scripting Editor, and Map Local Editor, ... rather than each one has their own setting.

**Comment by webuniverseio** [comment]
True. This is how I think about it, today there are 2 categories
- UI sizes, which are a bit different depending on a section 
- customizable sizes which are for user to control and are in "safe" to increase areas. 
 
To improve the experience of UI sizes which are small on hi-res displays but still have a predictable outcome it might make sense to introduce a pre-defined (hardcoded) setting which changes scale or maybe overrides configurations for UIs in such a way where you would know and test with certainty the size of the "hi-res" UI. Benefit is that it will be easier to maintain, downside is that it may not work for all users (some may want slightly smaller or slightly larger).

**Comment by 350d** [comment]
Hello! Please add font settings for Scripting console. I have checked `Use Monotype Font` in app settings but scripting console didn't respect this settings.

**Comment by NghiaTranUIT** [comment]
@350d it's fixed on this beta build: https://github.com/ProxymanApp/Proxyman/issues/2082#issuecomment-2227218566
